mne/beamformer/_lcmv.py

```diff
@@ -24,6 +24,16 @@
 from ..channels.channels import _contains_ch_type
 
 
+def _deprecate_picks(data_obj, picks):
+    if picks is not None:
+        warn('Specifying picks is deprecated and will be removed in 0.16. '
+             'Specifying the selection of channels in info and setting picks '
+             'to None will remove this warning.',
+             DeprecationWarning)
+
+        data_obj.info = pick_info(data_obj.info, picks)
```

Comment:
This is weird, it only modifies `info` and not `.data` (e.g., in the case of Evoked). This can instead be:
```
data_obj.pick_channels([data_obj.ch_names[pick] for pick in picks]
```